book/src/stake-delegation-and-rewards.md

```diff
@@ -195,3 +200,106 @@ stake.
 ## Example Callflow
 
 <img alt="Passive Staking Callflow" src="img/passive-staking-callflow.svg" class="center"/>
+
+## Staking Rewards
+
+The specific mechanics and rules of the validator rewards regime is outlined
+here.  Rewards are earned by delegating stake to a validator that is voting
+correctly.  Voting incorrectly exposes that validator's stakes to
+[slashing](staking-and-rewardsg.md).
+
+### Basics
+
+The network pays rewards from a portion of network [inflation](inflation.md).
+The number of lamports available to pay rewards for an epoch is fixed and
+must be evenly divided among all staked nodes according to their relative stake
+weight and pariticpation.  The weighting unit is called a
+[point](terminology.md#point).
+
+Rewards for an epoch are not available until the end of that epoch.
+
+At the end of each epoch, the total number of points earned during the epoch is
+summed and used to divide the rewards portion of epoch inflation to arrive at a
+point value.  This value is recorded in the bank in a
+[sysvar](terminology.md#sysvar) that maps epochs to point values.
+
+During redemption, the stake program counts the points earned by the stake for
+each epoch, multiplies that by the epoch's point value, and transfers lamports in
+that amount from a rewards account into the stake and vote accounts according to
+the vote account's commission setting.
```

Comment:
Does this mean that the validator doesn't earn commission until the staker withdraws?

Reply:
no, validators earn as they go.